src/libraries/System.Private.CoreLib/src/System/Threading/NativeRuntimeEventSource.PortableThreadPool.cs

```diff
@@ -8,11 +8,18 @@
 
 namespace System.Diagnostics.Tracing
 {
-    // This is part of the NativeRuntimeEventsource for Mono, which is the managed version of the Microsoft-Windows-DotNETRuntime provider.
-    // and contains the implementation of ThreadPool events.
+    // This is part of the NativeRuntimeEventsource for Mono|CoreRT, which is the managed version of the Microsoft-Windows-DotNETRuntime provider.
+    // and contains the implementation of ThreadPool events. Since file is shared between Mono|CoreRT it is kept in shared part of SPC.
     // To look at CoreCLR implementation of these events, refer to NativeRuntimeEventSource.PortableThreadPool.CoreCLR.cs.
     internal sealed partial class NativeRuntimeEventSource : EventSource
     {
+        // On Mono|CoreRT, we don't have these keywords defined from the genRuntimeEventSources.py, so we need to manually define them here.
+        public class Keywords
```

Comment:
nit: should be static

Reply:
Didn't want to deviate from what gets auto generated in runtime\artifacts\obj\coreclr\Eventing\x64\Debug\NativeRuntimeEventSource.CoreCLR.cs:

```
public class Keywords
        {
            public const EventKeywords GCKeyword = (EventKeywords)0x1;
            ....
```

so just followed along in above.